apps/www/lib/translations.json

```
@@ -3570,7 +3570,7 @@
     },
     {
       "key": "memberships/latestPeriod/renew/true/autoPay/false",
-      "value": "Bezahlt bis am {formattedEndDate}"
+      "value": "Erneuerung vor dem {formattedEndDate} nötig"
```

Vielleicht wäre das etwas präzisier und eindeutiger?
```suggestion
      "value": "Erneuerung am {formattedEndDate} fällig"
```

---

Jonas stimmt womit ich das Quorum verloren habe. Übernehme.